Domains/2-DisciplinePhysical/Civil/CivilUnits/Released/CivilUnits.01.00.00.ecschema.xml

```diff
@@ -0,0 +1,40 @@
+    <KindOfQuantity typeName="STATION" displayLabel="Civil Station" description="Linear referencing measurements along alignments for roads, railways, pipes, and linear infrastructure" persistenceUnit="u:M" presentationUnits="f:StationZ_1000_3[u:M];f:StationZ_100_2[u:FT];f:StationZ_100_2[u:US_SURVEY_FT];f:DefaultRealU(2)[u:M];f:DefaultRealU(2)[u:FT];f:DefaultRealU(2)[u:US_SURVEY_FT]" relativeError="0.0001"/>
+    <KindOfQuantity typeName="VELOCITY" displayLabel="Civil Velocity" description="Speed measurements for vehicles, water flow, and hydraulic velocity calculations" persistenceUnit="u:M_PER_SEC" presentationUnits="f:DefaultRealU(2)[u:KM_PER_HR];f:DefaultRealU(2)[u:MPH]" relativeError="0.0001"/>
+    <KindOfQuantity typeName="VOLUME" displayLabel="Civil Volume" description="Volumetric measurements for earthwork, storage facilities, and material quantities" persistenceUnit="u:CUB_M" presentationUnits="f:DefaultRealU(2)[u:CUB_M];f:DefaultRealU(2)[u:CUB_FT]" relativeError="0.0001"/>
+    <KindOfQuantity typeName="PERCENTAGE" displayLabel="Civil Percentage" description="Percentage values for impervious area calculations, hydraulic efficiency, inlet capture rates, storage properties, and positioning along linear elements" persistenceUnit="u:DECIMAL_PERCENT" presentationUnits="f:DefaultRealU(2)[u:PERCENT];f:DefaultRealU(4)[u:DECIMAL_PERCENT]" relativeError="0.0001"/>
+    <KindOfQuantity typeName="SIDE_SLOPE" displayLabel="Civil Side Slope" description="Horizontal-to-vertical slope ratios for embankments, cuts, and channel side slopes" persistenceUnit="u:M_HORIZONTAL_PER_M_VERTICAL" presentationUnits="f:DefaultRealU(1)[u:HORIZONTAL_PER_VERTICAL]" relativeError="0.0001"/>
+    <KindOfQuantity typeName="INTENSITY_INFILTRATION" displayLabel="Civil Intensity/Infiltration" description="Rainfall intensity and soil infiltration rate measurements for hydrologic and drainage design" persistenceUnit="u:M_PER_SEC" presentationUnits="f:DefaultRealU(2)[u:MM_PER_HR];f:DefaultRealU(2)[u:IN_PER_HR]" relativeError="0.0001"/>
+    <KindOfQuantity typeName="RATIONAL_COEFFICIENT" displayLabel="Civil Rational Coefficient" description="Runoff coefficient for the Rational Method in stormwater drainage calculations" persistenceUnit="u:COEFFICIENT" presentationUnits="f:DefaultRealU(2)[u:COEFFICIENT]" relativeError="0.0001"/>
+    <KindOfQuantity typeName="MANNINGS_N" displayLabel="Civil Manning's n" description="Manning's roughness coefficient for open channel and pipe flow calculations" persistenceUnit="u:COEFFICIENT" presentationUnits="f:DefaultRealU(3)[u:COEFFICIENT]" relativeError="0.0001"/>
+    <KindOfQuantity typeName="HAZEN_WILLIAMS_C" displayLabel="Civil Hazen-William's C" description="Hazen-Williams C coefficient for pressurized pipe flow and water distribution system calculations" persistenceUnit="u:COEFFICIENT" presentationUnits="f:DefaultRealU(0)[u:COEFFICIENT]" relativeError="0.0001"/>
```

Comment:
for koqs that represent coefficients, do you want to show a unit at all?  If not then use DefaultReal rather than DefaultRealU.